Document granular run notification triggers (#3248)
hashicorp/web-unified-docs: content/terraform-docs-common/docs/cloud-docs/api-docs/changelog.mdx

Keep track of changes to the API for HCP Terraform and Terraform Enterprise.

## 2026-08-24

> - Notification configurations support the following granular run triggers: `run:pending_apply_approval`, `run:cost_estimated`, `run:policy_override_required`, `run:policies_checked`, `run:run_errored`, `run:run_canceled`, and `run:policy_soft_failed`. Generic notification payloads report the subscribed trigger in the `trigger` field. The `run:needs_attention` and `run:errored` triggers are deprecated, but the API continues to accept them for backward compatibility. Refer to the [workspace notification configurations API reference](/terraform/cloud-docs/api-docs/notification-configurations/workspace#notification-triggers) for more information.

Reviewer: ```suggestion
- Replaced `run:needs_attention` and `run:errored` run triggers in notification configurations with more granular run triggers. Refer to the [project notification configurations API reference](/terraform/cloud-docs/api-docs/notification-configurations/project#notification-triggers) and [workspace notification configurations API reference](/terraform/cloud-docs/api-docs/notification-configurations/workspace#notification-triggers)  for more information.
```

This is really detailed for a changelog.
